test/fixtures/manifest.json

```diff
@@ -1,14 +1,33 @@
 {
```

Comment:
This one was an accident. 

The one above could be removed if we only want to run the tests on files created by the generator.